modules/dns_nsupdate/dns_nsupdate_main.rb

```diff
@@ -69,6 +80,10 @@ def remove_ptr_record(ip)
       do_remove(ip, "PTR")
     end
 
+    def remove_cname_record(fqdn_alias)
```

Comment:
Bikeshedding: I don't like ```fqdn_alias```. ```fqdn``` is in line with other code but name might be less ambiguous.